challenges/shakespearian-insult-generator.js

```diff
@@ -17,3 +17,23 @@ var second_word = ["weather-bitten", "unchin-snouted", "toad-spotted", "tickle-b
 var third_word = ["wagtail", "whey-face", "vassal", "varlet", "strumpet", "skainsmate", "scut", "ratsbane", "pumpion", "puttock", "pignut", "pigeon-egg", "nut-hook", "mumble-news", "moldwarp", "miscreant", "minnow", "measle", "mammet", "malt-worm", "maggot-pie", "lout", "lewdster", "joithead", "hugger-mugger", "horn-beast", "hedge-pig", "harpy", "haggard", "gudgeon", "giglet", "fustilarian", "foot-licker", "flirt-gill", "flax-wench", "flap-dragon", "dewberry", "death-token", "codpiece", "coxcomb", "clotpole", "clack-dish", "canker-blossom", "bum-bailey", "bugbear", "boar-pig", "bladder", "barnacle", "baggage", "apple-john"];
 
 // YOUR CODE HERE
+
+
+
+
+var randomItemFirst = first_word[Math.floor(Math.random()*first_word.length)];
+
+var randomItemSecond = second_word[Math.floor(Math.random()*second_word.length)];
+
+var randomItemThird = third_word[Math.floor(Math.random()*third_word.length)];
+
+
+
+var new_words = [randomItemFirst, randomItemSecond, randomItemThird];
+
+
```

Comment:
I wish that this had gotten to the stage of putting those words into the sentence together, but this is a good start!